build/common.sh

```diff
@@ -85,22 +85,35 @@ readonly KUBE_RSYNC_PORT="${KUBE_RSYNC_PORT:-}"
 # mapped to KUBE_RSYNC_PORT via docker networking.
 readonly KUBE_CONTAINER_RSYNC_PORT=8730
 
+# These are the default versions (image tags) for their respective base images.
+readonly __default_debian_iptables_version=buster-v1.5.0
+readonly __default_go_runner_version=buster-v2.3.1
+
```

Comment:
should we add one for setcap?

Reply:
In order for it to matter, bazel would have to be using it to apply capabilities to the binary artifact

Reply:
Got it!